assembl/views/api2/discussion.py

```diff
@@ -553,13 +560,13 @@ def extract_taxonomy_csv(request):
         else:
             qualify_by_action = "no qualify by action"
         owner_of_the_message = db.query(m.User).filter(m.User.id == query.creator_id).first().name
-        published_on = str(query.creation_date)
+        published_on = str(query.creation_date)[:19]
```

Comment:
I think we need to format the date before. It's not clean.

Reply:
yes, if `query.creation_date` is a `datetime.datetime`, use python tools to format the date : https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior